.mocharc.js

```diff
@@ -0,0 +1,16 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// Reuse the mocha config from `@loopback/cli`
+const config = require('./packages/cli/.mocharc.js');
+
+// Set max listeners to 16 for testing to avoid the following warning:
+// (node:11220) MaxListenersExceededWarning: Possible EventEmitter
+// memory leak detected. 11 SIGTERM listeners added to [process].
+// Use emitter.setMaxListeners() to increase limit
+// It only happens when multiple app instances are started but not stopped
```

Comment:
I think it's a problem in our test suite when we start apps but don't stop them. Instead of suppressing the helpful warning, can we please fix our tests to always stop the applications started?

Reply:
I am running Node.js 14.4.0 and haven't encountered `MaxListenersExceededWarning`. Could you please double check that this is still a problem? Is it perhaps specific to older versions of Node or a timing conditions on CI build servers?